Lesson23/Kubra/filter.js

```diff
@@ -352,6 +358,9 @@ const studentsExercise9 = [
 		grade: 'B-',
 	},
 ];
+const studentsExercise9Result = studentsExercise9.filter(student => student.grade[0] !== "D");
```

Comment:
Nice, this is actually a smart way to avoid writing 3 different case (D, D+, D-). However, F is also a grade that is lover than C. Event if it's not present in the test data, please add it as well!